### DISCUSSION TITLE
how to upgrade to latest version?

### DISCUSSION THREAD

**Discussion by brighthorizon** [sorayuki/obs-multi-rtmp#150] [Q&A]
i have an older version of plugin but I do not see any instructions for upgrading to latest version. Does anyone know how?

**Comment by iandouglas** [comment]
When I downloaded the latest .exe file, Windows detects a virus. Instead I downloaded the "obs-multi-rtmp-portable.zip" archive, and copied its contents into my OBS installation folder.

Inside the .zip folder you'll find a "data" folder and "obs-plugins" folder.

Unpack them as-is into your OBS installation folder, where you'll find a matching set of folders. (my OBS installation had a 'bin' folder, a 'data' folder and an 'obs-plugins' folder.

Start up OBS, and you should be good to configure it from there.

**Reply by brighthorizon** [reply]
great i will try that thank you

**Reply by iskconjaipur** [reply]
> > When I downloaded the latest .exe file, Windows detects a virus. Instead I downloaded the "obs-multi-rtmp-portable.zip" archive, and copied its contents into my OBS installation folder.
> > Inside the .zip folder you'll find a "data" folder and "obs-plugins" folder.
> > Unpack them as-is into your OBS installation folder, where you'll find a matching set of folders. (my OBS installation had a 'bin' folder, a 'data' folder and an 'obs-plugins' folder.
> > Start up OBS, and you should be good to configure it from there.
> 
> Hey, Is there a problem with the latest 0.2.8.. Either unzipping or using the executable doesn't upgrade it. The unzip says the old version is installed. needs to remove the dll. and the exe just uninstalls the older version. That's it. Does nothing else.
@sorayuki 
![image](https://user-images.githubusercontent.com/64528003/159178077-1a5812cc-c5a5-43a1-bada-f41908cb0d2e.png)
Using the installer, always asks to uninstall, even when running multiple times.
![image](https://user-images.githubusercontent.com/64528003/159178082-d2efee39-ff07-478e-8bb2-6919ef752036.png)
when unzipping, into the root obs directory, this is the error it puts out.
